work17/index.js

```diff
@@ -0,0 +1,191 @@
+"use strict";
+const REQUEST_URL = "https://mocki.io/v1/d4a57e5a-8c84-4fee-aa05-70a1791c0d86";
+
+// 間違っているURLの場合↓
+// const REQUEST_URL = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// 空の配列の場合↓
+// const REQUEST_URL = "https://mocki.io/v1/242b685f-a3d7-45a8-aeca-0376bd495b89";
+// 503エラーの場合↓
+// const REQUEST_URL = "https://httpstat.us/503";
+
+const wrapDiv = document.querySelector(".slider_wrap");
```

Comment:
IMO
It would be safer to grant and obtain an ID

Reply:
ID was safer and quicker to obtain.
Thank you.

commit: [769bf64](https://github.com/yasu-pro/js-handson/pull/22/commits/769bf64f16342cec45254281e1ab822738b8a620)